src/nemos/regularizer.py

```diff
@@ -279,10 +139,17 @@ class are defined in the `allowed_solvers` attribute.
         "LBFGSB",
     )
 
+    _default_solver = "GradientDescent"
+
     def __init__(
-        self, solver_name: str = "GradientDescent", solver_kwargs: Optional[dict] = None
+            self,
+            regularizer_strength: float = 1.0
     ):
-        super().__init__(solver_name, solver_kwargs=solver_kwargs)
+        super().__init__(regularizer_strength=regularizer_strength)
+
+    def penalized_loss(self, loss: Callable):
+        """Unregularized method does not add any penalty."""
+        return loss
 
 
 class Ridge(Regularizer):
```

Comment:
Ridge, as any other, should also have a `get_proximal_operator` method. Check the prox operator for ridge in the jaxopt codebase.